public/css/tags.css

```diff
@@ -1,57 +1,69 @@
 .tag {
-	background-color: var(--header);
-	font-size: 85%;
-	padding: 0.5em .75em;
-	margin: .25em .1em;
-	display: inline-block;
-	border: 1px solid var(--border-light);
-	transition: all .1s linear;
+  background-color: var(--header);
+  font-size: 85%;
+  padding: 0.5em 0.75em;
+  margin: 0.25em 0.1em;
+  display: inline-block;
+  border: 1px solid var(--border-light);
+  transition: all 0.1s linear;
 }
 
 .tag-sort-placeholder {
   height: 44px;
-	background-color: var(--bg-hover);
+  background-color: var(--bg-hover);
   margin-left: -15px;
   margin-right: -15px;
 }
 
 .tag-sort-helper {
-	background-color: var(--bg);
-	border: 1px solid var(--border-light);
+  background-color: var(--bg);
+  border: 1px solid var(--border-light);
 }
 
-.tag-color,
-.tag-color .tdname,
-.tag-color > a {
-  color: var(--bg) !important;
```

Comment:
Removed this for a couple reason:
1. The text color needs to be based on the tag background color not the theme in order to have good contrast
2. I found no usages of tdname
3. The only time a tag renders as a link is in the CubeOverview and there it doesn't have tag-color class above it. Thus redundant